webpack.config.js

```diff
@@ -42,15 +42,11 @@ var webpackSettings = {
     },
 }
 
-var settings = webpackSettings[yargs.argv.set];
-if (settings == undefined) {
-    settings = webpackSettings['dev'];
-}
-
 module.exports = function (env) {
-    var settings = webpackSettings[env.mode];
+    const mode = (env && env.mode) || "prod";
+    var settings = webpackSettings[mode];
     if (settings == undefined) {
-        settings = webpackSettings['dev'];
+        settings = webpackSettings['prod'];
```

Comment:
Are these changes intended to check in, or just things leftover from dev/test?

Reply:
Planning on checking them in. Default build settings should be prod unless otherwise specified as a general good practice. I think the build, test and dev tasks need to be fixed up a bit.

Reply:
Feel free to jot these EE comments down as ADO items!